app/views/catalog/_form_basic_info.html.haml

```diff
@@ -6,65 +6,72 @@
     = render(:partial=>'layouts/ae_tree_select')
 
   %h3=_('Basic Info')
-  %table.style1
-    %tr
-      %td.key=_('Name / Description')
-      %td
-        = text_field_tag("name",
+  %form.form-horizontal
+    .form-group
+      %label.col-sm-2
```

Comment:
All labels should be: %label.col-sm-2.control-label (makes labels right-justified and bolded)